third_party/zstd.BUILD

```diff
@@ -0,0 +1,85 @@
+cc_library(
+    name = "zstd",
+    srcs = [
+        "lib/common/debug.c",
+        "lib/common/entropy_common.c",
+        "lib/common/error_private.c",
+        "lib/common/fse_decompress.c",
+        "lib/common/pool.c",
+        "lib/common/threading.c",
+        "lib/common/xxhash.c",
+        "lib/common/zstd_common.c",
+        "lib/compress/fse_compress.c",
+        "lib/compress/hist.c",
+        "lib/compress/huf_compress.c",
+        "lib/compress/zstd_compress.c",
+        "lib/compress/zstd_compress_literals.c",
+        "lib/compress/zstd_compress_sequences.c",
+        "lib/compress/zstd_compress_superblock.c",
+        "lib/compress/zstd_double_fast.c",
+        "lib/compress/zstd_fast.c",
+        "lib/compress/zstd_lazy.c",
+        "lib/compress/zstd_ldm.c",
+        "lib/compress/zstd_opt.c",
+        "lib/compress/zstdmt_compress.c",
+        "lib/decompress/huf_decompress.c",
+        "lib/decompress/huf_decompress_amd64.S",
+        "lib/decompress/zstd_ddict.c",
+        "lib/decompress/zstd_decompress.c",
+        "lib/decompress/zstd_decompress_block.c",
+        "lib/dictBuilder/cover.c",
+        "lib/dictBuilder/divsufsort.c",
+        "lib/dictBuilder/fastcover.c",
+        "lib/dictBuilder/zdict.c",
+    ],
+    hdrs = [
+        "lib/common/allocations.h",
+        "lib/common/bits.h",
+        "lib/common/bitstream.h",
+        "lib/common/compiler.h",
+        "lib/common/cpu.h",
+        "lib/common/debug.h",
+        "lib/common/error_private.h",
+        "lib/common/fse.h",
+        "lib/common/huf.h",
+        "lib/common/mem.h",
+        "lib/common/pool.h",
+        "lib/common/portability_macros.h",
+        "lib/common/threading.h",
+        "lib/common/xxhash.h",
+        "lib/common/zstd_deps.h",
+        "lib/common/zstd_internal.h",
+        "lib/common/zstd_trace.h",
+        "lib/compress/clevels.h",
+        "lib/compress/hist.h",
+        "lib/compress/zstd_compress_internal.h",
+        "lib/compress/zstd_compress_literals.h",
+        "lib/compress/zstd_compress_sequences.h",
+        "lib/compress/zstd_compress_superblock.h",
+        "lib/compress/zstd_cwksp.h",
+        "lib/compress/zstd_double_fast.h",
+        "lib/compress/zstd_fast.h",
+        "lib/compress/zstd_lazy.h",
+        "lib/compress/zstd_ldm.h",
+        "lib/compress/zstd_ldm_geartab.h",
+        "lib/compress/zstd_opt.h",
+        "lib/compress/zstdmt_compress.h",
+        "lib/decompress/zstd_ddict.h",
+        "lib/decompress/zstd_decompress_block.h",
+        "lib/decompress/zstd_decompress_internal.h",
+        "lib/dictBuilder/cover.h",
+        "lib/dictBuilder/divsufsort.h",
+        "lib/zdict.h",
+        "lib/zstd.h",
+        "lib/zstd_errors.h",
+    ],
+    copts = ["-fpic"],
```

Comment:
Can we remove this?

To me this is something we should enable as part of a build configuration instead of a specific library (we probably aren't linking this shared anyway).